apps/payments/next/app/[locale]/[offeringId]/[interval]/landing/route.ts

```diff
@@ -149,7 +149,7 @@ export async function GET(
   try {
     try {
       const metricsFlow = await getMetricsFlowAction();
-      redirectToUrl.searchParams.set('flowId', metricsFlow.flowId);
+      redirectToUrl.searchParams.set('flow_id', metricsFlow.flowId);
```

Comment:
nitpick: Could the other params also be updated to snake case?

Reply:
Yes, though I think we may want to leave `spVersion` as-is, as it's likely used in too many QA/verification links that have been saved as plaintext to reasonably update (and it'll be removed eventually)